lamindb_setup/core/_hub_core.py

```diff
@@ -137,10 +138,14 @@ def init_storage(
 
 
 def _init_storage(
-    ssettings: StorageSettings, auto_populate_instance: bool, client: Client
+    client: Client,
```

Comment:
`client` has to be the last argument I believe. Otherwise the `call_with_fallback_auth` logic doesn't work, does it?

Reply:
It doesn't matter at all, `client` is passed with the argument name.

Reply:
Ok!